packages/contracts/utils/etherscan.ts

```diff
@@ -1,6 +1,15 @@
 import networks from '../networks';
+import {
+  AllDeployments,
+  NameAndAddress,
+  CollectedProxyWithImplementation,
+} from '../types/etherscan';
+import {ChainConfig} from '@nomicfoundation/hardhat-verify/dist/src/types';
+import axios, {AxiosRequestConfig} from 'axios';
```

Comment:
Shouldn't `axios` be included as a `devDependency`?

Reply:
Added